src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -65,51 +59,59 @@ public final void setBridgeCredentials(SignIn bridgeCredentials) {
         List<Upload> retList = new ArrayList<>();
         String offsetKey = null;
 
+        ForWorkersApi workersApi = bridgeClientManager.getClient(ForWorkersApi.class);
         do {
+            
             final String temOffsetKey = offsetKey;
-            UploadList retBody = sessionHelper(() -> bridgeClientManager.getClient(ForWorkersApi.class).getUploadsInStudy(studyId,
-                    startDateTime, endDateTime, MAX_PAGE_SIZE, temOffsetKey).execute().body());
+            UploadList retBody = workersApi
+                    .getUploadsInStudy(studyId, startDateTime, endDateTime, MAX_PAGE_SIZE, temOffsetKey).execute()
+                    .body();
             retList.addAll(retBody.getItems());
             offsetKey = retBody.getOffsetKey();
-            // sleep a second
-            try {
-                Thread.sleep(THREAD_SLEEP_INTERVAL);
-            } catch (InterruptedException e) {
-                LOG.warn("The thread for get uploads was being interrupted.", e);
-            }
+            doSleep();
         } while (offsetKey != null);
 
         return retList;
     }
-
-    /**
-     * Helper method to save report for specified study with report id and report data
-     */
-    public void saveReportForStudy(String studyId, String reportId, ReportData reportData) throws IOException {
-        sessionHelper(() -> bridgeClientManager.getClient(ForWorkersApi.class).saveReportForStudy(studyId, reportId,
-                reportData).execute());
+    
+    public List<StudyParticipant> getParticipantsForStudy(String studyId, DateTime startDateTime, DateTime endDateTime)
+            throws IOException {
+        List<StudyParticipant> retList = new ArrayList<>();
+        
+        ForWorkersApi workersApi = bridgeClientManager.getClient(ForWorkersApi.class);
+        
+        int offset = 0;
+        do {
+            AccountSummaryList summaries = workersApi
+                    .getParticipantsInStudy(studyId, offset, PARTICIPANT_PAGE_SIZE, null, startDateTime, endDateTime)
+                    .execute().body();
+            for (AccountSummary summary : summaries.getItems()) {
+                StudyParticipant participant = workersApi.getParticipantInStudy(studyId, summary.getId()).execute().body();
+                retList.add(participant);
+                doSleep();
+            }
+            offset = (summaries.getOffsetBy() != null) ? summaries.getOffsetBy().intValue() : -1;
+        } while(offset > 0);
```

Comment:
Changed this to use Long, and convert to Integer, because one of these calls returns Long, and the other takes Integer. These should be consistent, I filed a JIRA on this: https://sagebionetworks.jira.com/browse/BRIDGE-1878